devel/pcre/Portfile

```diff
@@ -85,6 +85,16 @@ platform darwin 8 {
     }
 }
 
+if {${subport} eq "pcre2" && ${os.platform} eq "darwin"} {
+    # This is specific to Intel and libstdc++.
+    # https://trac.macports.org/ticket/67477
+    # https://trac.macports.org/ticket/69073
+    if {${configure.build_arch} in "i386 x86_64" && ${configure.cxx_stdlib} ne "libc++"} {
+        configure.args-replace \
+                    --enable-jit --disable-jit
+    }
+}
+
```

Comment:
Does this only apply to older versions of MacOS that actually have the problem?

Reply:
@pmetzger It may be non-trivial to build anything against `lidstdc++` on newer systems. It is certainly not a default config, whether in Macports or anywhere on macOS, AFAIK.
I can try now, but generally speaking I could not even make Macports on Sonoma work with `lidstdc++` properly. It seems that Apple headers are just incompatible with it.

Reply:
P. S. To be clear, I will not object adding an additional condition for < 10.7, but I think it is redundant at the very least (one has to hack Macports or macOS to use `libstdc++` on 10.7 or later).

Reply:
It won't hurt to have the redundancy.

Reply:
Sure, let me add it now.